src/views/HomepageView/Calendar/index.vue

```diff
@@ -0,0 +1,350 @@
+            linear-gradient(225deg, $color-christmas-red 25%, transparent 25%) -25px 0,
+            linear-gradient(315deg, $color-christmas-red 25%, transparent 25%),
+            linear-gradient(45deg, $color-christmas-red 25%, transparent 25%);
+          background-size: 50px 50px;
+          background-color: $color-christmas-red-dark;
+          color: $color-white;
+        }
+      }
+
+      // CENTERED ICON
+      #cell-13 :deep(.calendar-cell) {
+        background: $color-christmas-green;
+
+        .icon {
+          color: $color-christmas-yellow;
+
+          svg {
+            fill: $color-christmas-yellow;
+          }
+        }
+      }
+
+      // ICON
+      #cell-14 :deep(.calendar-cell) {
+        background-color: $color-christmas-yellow;
+        color: $color-background-light;
+      }
+
+      // DOTS
+      #cell-15 :deep(.calendar-cell) {
+        background-color: $color-christmas-red;
+        background-image: radial-gradient($color-christmas-yellow-dark 12%, transparent 0),
+          radial-gradient($color-christmas-yellow-dark 12%, transparent 0);
+        background-size: 20px 20px;
+        background-position: 0 0, (20px / 2) (20px / 2);
+        color: $color-white;
+      }
+
+      // DOTS
+      #cell-16 :deep(.calendar-cell) {
+        background-color: $color-background-light;
+        background-image: radial-gradient($color-background-very-light 12%, transparent 0),
+          radial-gradient($color-background-very-light 12%, transparent 0);
+        background-size: 20px 20px;
+        background-position: 0 0, (20px / 2) (20px / 2);
+        color: $color-white;
+      }
+
+      // ICON
+      #cell-17 :deep(.calendar-cell) {
+        background-color: $color-background-light;
+        color: $color-white;
+      }
+
+      // ZIGZAG
+      #cell-20 {
+        &.calendar-cell-container {
+          grid-area: 2 / 7 / 4 / 7;
+        }
+        :deep(.calendar-cell) {
+          background: linear-gradient(135deg, $color-christmas-green 25%, transparent 25%) -25px 0,
+            linear-gradient(225deg, $color-christmas-green 25%, transparent 25%) -25px 0,
+            linear-gradient(315deg, $color-christmas-green 25%, transparent 25%),
+            linear-gradient(45deg, $color-christmas-green 25%, transparent 25%);
+          background-size: 50px 50px;
+          background-color: $color-christmas-green-dark;
+          color: $color-background-light;
+        }
+      }
+
+      // DOTS
+      #cell-21 {
+        &.calendar-cell-container {
+          grid-area: 1 / 2 / 1 / 4;
+        }
+        :deep(.calendar-cell) {
+          background-color: $color-christmas-red;
+          background-image: radial-gradient($color-christmas-red-dark 12%, transparent 0),
+            radial-gradient($color-christmas-red-dark 12%, transparent 0);
+          background-size: 20px 20px;
+          background-position: 0 0, (20px / 2) (20px / 2);
```

Comment:
DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div(20px, 2)